src/BlazorDB/IndexDbManager.cs

```diff
@@ -300,7 +299,41 @@ public async Task<IList<TRecord>> ToArray<TRecord>(string storeName)
 
             return default;
         }
-        
+       
+        /// <summary>
+        /// Filter a store on an indexed value 
+        /// </summary>
+        /// <param name="storeName">The name of the store to retrieve the records from</param>
+        /// <param name="key">index field name to filter on</param>
+        /// <param name="value">filter's value</param>
+        /// <returns></returns>
+        public async Task<IList<TRecord>> Where<TRecord>(string storeName, string key, object value)
+        {
+            var trans = GenerateTransaction(null);
+
+            try
+            {
+                
+                return await CallJavascript<IList<TRecord>>(IndexedDbFunctions.WHERE,  trans, DbName, storeName, FormatConditionJsonString(key, value));
+            }
+            catch (JSException jse)
+            {
+                RaiseEvent(trans, true, jse.Message);
+            }
+
+            return default;
+        }
+
+        private string FormatConditionJsonString(string key, object value)
+        {
+            if (value is string)
+            {
+                return $"{{ \"{key}\": \"{value}\" }}";
+            }
+            
+            return $"{{ \"{key}\": {value} }}";
+        }
```

Comment:
I have to think about this, the condition for `is string` is good, but if the `object` is not something like an `int`, we might want to serialize it.

Or, you pass the key/value to Javascript and have that worry about the serialization

Reply:
Yeah, I'm not very confident about this code, the thing is we have to generate some JSON to be interpreted on javascript side, the keys are dynamic, so it was "easier" to generate it as a simple string on c# side.

I will dig about this subject a bit.

Reply:
@nwestfall I changed the way the filter object is generated, delegating the creation of the json to javascript part, this put the responsability to form the correct filter on js side, closer to dexie.
In an other way, we let c# serialize the IndexFilterValue, so we don't care who need quotes or not anymore.

Let me know what you think about it.